CMakeLists.txt

```diff
@@ -1,16 +1,41 @@
-project(libhausbus)
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
```

Comment:
CMP0048 need to be set to NEW for project(.. VERSION ..) 
Will upload a patch, so that VERSION 2.8 is enough.